### PR TITLE
Add docs of clipboard

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Re- https://github.com/appium/appium/pull/10450

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/device/clipboard/get-clipboard.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  >
+    Get the content of the system clipboard
+
+example_usage:
+  java:
+    |
+      driver.getClipboardText();
+  python:
+    |
+      self.driver.get_clipboard_text()
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/get-clipboard.yml]: `get(set)_clipboard_text` is just a specific case of the generic `get(set)_clipboard` call, which only works with plain text. The same in Java. There are also some specific methods in Java for setting images or URLs in iOS, but these generic ones exist everywhere

File: commands-yml/commands/device/clipboard/set-clipboard.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      type: String
+      description: The actual clipboard content.
+    - name: contentType
+      type: String
+      description: The type of the content to get. Plaintext, Image, URL. Android supports only plaintext.
+    - name: label
+      type: String
+      description: Clipboard data label for Android.
+  response:
+    - type: string
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/set-clipboard.yml]: `String` or `string`?

File: commands-yml/commands/device/clipboard/set-clipboard.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+endpoint:
+  url: /wd/hub/session/:session_id/appium/device/set_clipboard
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
+    - name: content
+      type: String
+      description: The actual clipboard content.
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/set-clipboard.yml]: I assume it makes sense to mention that the content should be base64-encoded

File: commands-yml/commands/device/clipboard/set-clipboard.yml [hunk 4]
```diff
@@ -8,9 +8,11 @@ description:
 example_usage:
   java:
     |
+      driver.setClipboard('happy testing');
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/set-clipboard.yml]: setClipboard in Java expects bytearray as the first argument and contentType to be set explicitly

File: commands-yml/commands/device/clipboard/set-clipboard.yml [hunk 5]
```diff
@@ -8,9 +8,11 @@ description:
...
       driver.setClipboardText('happy testing');
   python:
     |
+      self.driver.set_clipboard('happy testing')
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/set-clipboard.yml]: setClipboard in Java expects contentType to be set explicitly

File: commands-yml/commands/device/clipboard/set-clipboard.yml (hunk 4 above)
Comment (mykola-mokhnach) [reply commands-yml/commands/device/clipboard/set-clipboard.yml]:
```
    /**
     * Set the content of device's clipboard.
     *
     * @param contentType one of supported content types.
     * @param base64Content base64-encoded content to be set.
     */
    default void setClipboard(ClipboardContentType contentType, byte[] base64Content) {
```

File: docs/en/commands/device/clipboard/get-clipboard.md [hunk 6]
```diff
@@ -5,12 +5,14 @@ Get the content of the system clipboard
 
 ```java
 // Java
+driver.getClipboard();
```
Comment (mykola-mokhnach) [inline docs/en/commands/device/clipboard/get-clipboard.md]: this requires contentType to be set in Java

File: docs/en/commands/device/clipboard/set-clipboard.md [hunk 7]
```diff
@@ -5,12 +5,14 @@ Set the content of the system clipboard
 
 ```java
 // Java
+driver.setClipboard('happy testing');
```
Comment (mykola-mokhnach) [inline docs/en/commands/device/clipboard/set-clipboard.md]: same here

File: commands-yml/commands/device/clipboard/get-clipboard.yml [hunk 8]
```diff
@@ -65,11 +67,8 @@ endpoint:
...
   json_parameters:
-    - name: content
-      type: String
-      description: The actual clipboard content.
     - name: contentType
-      type: String
+      type: string
       description: The type of the content to get. Plaintext, Image, URL. Android supports only plaintext.
   response:
     - type: string
```
Comment (mykola-mokhnach) [inline commands-yml/commands/device/clipboard/get-clipboard.yml]: the fact that the response string is base64-encoded depends on the called method. If we call the generic getClipboard then the response is encoded, but the getClipboard text shortcut performs the decoding for us and returns a plan text.

**Review by imurchie — APPROVED** [review]

**Pull request merged**